app/catalog/catalog_cli.go

```diff
@@ -156,6 +157,37 @@ var catalogCmdDef = &cli.Command{
 				util.CmdMiddlewareTracingSpan,
 			),
 		},
+		{
+			Name:  "serve",
+			Usage: "Generates html and serves",
+			Action: util.ChainCmdMiddleware(cmdServe,
+				util.CmdMiddlewareLogging,
+				util.CmdMiddlewareTracingConfig,
+				util.CmdMiddlewareTracingSpan,
+			),
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "output",
+					Aliases: []string{"o"},
+					Usage:   "Output path for HTML generation",
+				},
+				&cli.StringFlag{
+					Name:  "url-prefix",
+					Usage: "URL prefix for links within generated HTML",
+					Value: "/",
+				},
```

Comment:
Let's drop this too, per comment below about making this as works-by-default as possible.  We control the whole mux that's processing these URLs and not supporting configuring it, so this option doesn't even make sense in that story.